src/current/v23.1/monitoring-and-alerting.md

```diff
@@ -189,6 +189,685 @@ If you rely on external tools for storing and visualizing your cluster's time-se
+| `config.numVoters`             | `0`                                                          | The [number of voting replicas](configure-replication-zones.html#num_voters) for the range.                                    |
+| `config.constraints`           | `[...]`                                                      | The [constraints](configure-replication-zones.html#constraints) for the range.                                                 |
+| `config.voterConstraints`      | `[...]`                                                      | The [voting replica constraints](configure-replication-zones.html#voter_constraints) for the range.                            |
+| `config.leasePreferences`      | `[...]`                                                      | The [lease preferences](configure-replication-zones.html#lease_preferences) for the range.                                     |
+| `config.rangefeedEnabled`      | `false`                                                      | Whether [rangefeeds](create-and-configure-changefeeds.html#enable-rangefeeds) are enabled for this range.                      |
+| `config.excludeDataFromBackup` | `false`                                                      | Whether this range's data should be excluded from [backups](backup.html).                                                      |
+
+#### Examples
+
+- [Replication status - normal](#replication-status-normal)
+- [Replication status - constraint violation](#replication-status-constraint-violation)
+- [Replication status - under-replicated ranges](#replication-status-under-replicated-ranges)
+- [Replication status - ranges in critical localities](#replication-status-ranges-in-critical-localities)
+
+##### Replication status - normal
+
+The following example assumes you are running a fresh, local [`cockroach demo`](cockroach-demo.html) cluster started using the following command:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+cockroach demo --geo-partitioned-replicas --insecure
+~~~
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+curl -X POST http://localhost:8080/_status/critical_nodes
+~~~
+
+~~~ json
+{
+  "criticalNodes": [
+  ],
+  "report": {
+    "underReplicated": [
+    ],
+    "overReplicated": [
+    ],
+    "violatingConstraints": [
+    ],
+    "unavailable": [
+    ],
+    "unavailableNodeIds": [
+    ]
+  }
+}
+~~~
+
+{{site.data.alerts.callout_info}}
+You may have to wait a few minutes after starting the demo cluster before getting the 'all clear' output above. This can happen because it takes time for [replica movement](architecture/replication-layer.html) to occur to meet the constraints given by the [zone configurations](configure-replication-zones.html) set by the [`--geo-partitioned-replicas` flag](cockroach-demo.html#geo-partitioned-replicas).
```

Comment:
```suggestion
You may have to wait a few minutes after starting the demo cluster before getting the 'all clear' output above. This can happen because it takes time for [replica movement](architecture/replication-layer.html) to occur in order to meet the constraints given by the [zone configurations](configure-replication-zones.html) set by the [`--geo-partitioned-replicas` flag](cockroach-demo.html#geo-partitioned-replicas).
```

Reply:
Updated!